go/consensus/tendermint/full/full.go

```diff
@@ -600,6 +601,27 @@ func (t *fullService) lazyInit() error {
 		return err
 	}
 
+	// Convert addresses and public keys to Tendermint form. Converted IDs need to be lowercase
```

Comment:
Probably no need for the "Converted IDs..." part of the comment as it is already explained in the function doc.